lmos-runtime-core/src/main/kotlin/ai/ancf/lmos/runtime/outbound/LmosOperatorAgentRegistry.kt

```diff
@@ -55,7 +61,22 @@ class LmosOperatorAgentRegistry(private val lmosRuntimeProperties: LmosRuntimePr
         channelId: String,
         subset: String?,
     ): RoutingInformation {
-        val urlString = "${lmosRuntimeProperties.agentRegistry.baseUrl}/apis/v1/tenants/$tenantId/channels/$channelId/routing"
+        val routingInformation =
+            lmosRuntimeTenantAwareCache.get(tenantId, ROUTES, "$channelId-${subset ?: DEFAULT_ROUTE}")
+                ?: queryOperatorForRoutes(tenantId, channelId, subset).also {
+                    lmosRuntimeTenantAwareCache.save(tenantId, ROUTES, "$channelId-${it.subset ?: DEFAULT_ROUTE}", it, lmosRuntimeConfig.cache.ttl)
+                }
+
+        return routingInformation
+    }
+
+    private suspend fun queryOperatorForRoutes(
+        tenantId: String,
+        channelId: String,
+        subset: String?,
```

Comment:
Please remove the subset here.

Reply:
added due to reason - https://github.com/lmos-ai/lmos-runtime/pull/68#discussion_r1890579642

Reply:
@RobWin does this look fine or any change is required